event_sink_clickhouse/apps.py

```diff
@@ -11,3 +12,39 @@ class EventSinkClickhouseConfig(AppConfig):
     """
 
     name = 'event_sink_clickhouse'
+    verbose_name = "Event Sink ClickHouse"
+
+    plugin_app = {
+        PluginSettings.CONFIG: {
+            'lms.djangoapp': {
+                'production': {PluginSettings.RELATIVE_PATH: 'settings.production'},
+                'common': {PluginSettings.RELATIVE_PATH: 'settings.common'},
+            },
+            'cms.djangoapp': {
+                'production': {PluginSettings.RELATIVE_PATH: 'settings.production'},
+                'common': {PluginSettings.RELATIVE_PATH: 'settings.common'},
+            }
+        },
+        # Configuration setting for Plugin Signals for this app.
```

Comment:
I think we can remove these inline comments since the configurations is pretty self-explanatory